beatport_importer.user.js

```diff
@@ -62,6 +62,10 @@ function retrieveReleaseInfo(release_url) {
       catno: ProductDetail.catalog
     }
   );
+  
+  // Reload Playables if empty
+  if(!unsafeWindow.Playables.hasOwnProperty("tracks"))
+    eval($("#data-objects").text());
```

Comment:
Just evaluating this script again is not enough, because we copied `window` into `unsafeWindow` and evaluating updates `window` object but not `unsafeWindow`.

```js
if(!unsafeWindow.Playables.hasOwnProperty("tracks")) {
    eval($("#data-objects").text());
    unsafeWindow.Playables = window.Playables;
}
```
This can solve the issue.